### PR TITLE
Feature/test

### PR DESCRIPTION
**Pull request by Kwamay**
In this model, We .........

-[ ] Tested for the operate.js

-[ ] Tested for the calculate.js

### CODE REVIEW

File: src/logic/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import calculate from './calculate';
+
+describe('calculate function', () => {
+  it('should handle AC button', () => {
+    const result = calculate({ total: '10', next: '5', operation: '+' }, 'AC');
+    expect(result).toEqual({ total: null, next: null, operation: null });
+  });
+
+  it('should handle number buttons', () => {
```
Comment (Olamarx) [inline src/logic/calculate.test.js:9]: - [ ] I can see some of the functionalities & scenarios of the operate.js and calculate.js modules is being covered using Jest but you might have forgotten that you need to also need to make use of Jest snapshots for testing all the components, none of the components is tested. Remember to also make use of React Testing Library to simulate user interaction.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Readme file professional ✔️
- Test cases passed ✔️

## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly remeber to fix your MIT link too, it goes to 404 page at the moment.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bellom** [comment]
Dear Senior CR,

Please note: I think we have a glitch here, student commit pops up after the review.

image

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @Obote , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Clean code syntax
✅ ... Green linters
---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
